modules/installation-configuration-parameters.adoc

```diff
@@ -1223,6 +1223,54 @@ Additional Azure configuration parameters are described in the following table:
 |Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance. If instance type of compute machines support `Accelerated` networking, by default, the installer enables `Accelerated` networking, otherwise the default networking type is `Basic`.
 |`Accelerated` or `Basic`.
 
+|`compute.platform.azure.type`
+|Defines the azure instance type for compute machines.
+|String
+
+|`compute.platform.azure.zones`
+|The availability zones where the installation program creates compute machines.
+|String list
+
+|`controlPlane.platform.azure.type`
+|Defines the azure instance type for control plane machines.
+|String
+
+|`controlPlane.platform.azure.zones`
+|The availability zones where the installation program creates control plane machines.
+|String list
+
+|`platform.azure.defaultMachinePlatform.encryptionAtHost`
+|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption.
```

Comment:
```suggestion
|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached, and un-managed disks on the VM host. This parameter is not a prerequisite for user-managed server-side encryption.
```

Per IBM Style, make sure that all pronoun references, such as _This_ are clear: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references